tests/framework/db/mssql/CommandTest.php

```
@@ -719,6 +719,176 @@ public function testAlterColumn(): void
+            'The original unique constraint must remain after the failure.',
+        );
+        self::assertSame(
+            $originalUniques[0]->name,
+            $uniques[0]->name,
+            'The original unique constraint must be restored.',
+        );
+    }
+
+    public function testAlterColumnDoesNotCommitExternalTransaction(): void
+    {
+        $db = $this->getConnection();
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $db. Configured minimum length is 3. (undefined)

(ShortVariable)